tensorflow/training/docker/2.4/py3/cu110/Dockerfile.gpu

```diff
@@ -238,6 +244,22 @@ RUN  wget -nv https://github.com/rapidsai/rmm/archive/v${RMM_VERSION}.tar.gz \
   && rm -rf v${RMM_VERSION}.tar* \
   && rm -rf rmm-${RMM_VERSION}
 
+# Install AWS OFI NCCL plug-in
+RUN apt-get update && apt-get install -y autoconf
+RUN mkdir /tmp/efa-ofi-nccl \
+  && cd /tmp/efa-ofi-nccl \
+  && git clone https://github.com/aws/aws-ofi-nccl.git -b v$BRANCH_OFI \
+  && cd aws-ofi-nccl \
+  && ./autogen.sh \
+  && ./configure --with-libfabric=/opt/amazon/efa \
+  --with-mpi=/opt/amazon/openmpi \
+  --with-cuda=/usr/local/cuda \
+  --with-nccl=/usr/local --prefix=/usr/local \
+  && make \
+  NVCC_GENCODE="-gencode=arch=compute_70,code=sm_70 -gencode=arch=compute_75,code=sm_75 -gencode=arch=compute_80,code=sm_80 -gencode=arch=compute_80,code=compute_80" \
```

Comment:
Please confirm that these GPU archs are a subset of those supported by TF framework on this dockerfile.

Reply:
removed this, we added this to save build time to build image just for those archs, but I think we dont need this code as it is difficult to maintain

Reply:
We need to minimize DLC image sizes as well, and if constraining the list of supported archs does this for us, then we should prefer to constrain the list in some way or the other.